### DISCUSSION TITLE
What RFID reader for stop on remove?

### DISCUSSION THREAD

**Discussion by bergziege** [MiczFlor/RPi-Jukebox-RFID#1616] [Q&A] [answered]
Hi all, 
yesterday I build my first phoniebox with on/off shim, miniamp and neuftech and it works great!
BUT I had to learn the hard way that usb readers do not support the "remove to stop" feature which is the main feature I wanted to build the the box for.

So after reading the full manual I still have no clue what RFID do I need to support said feature?

There seem to be a few tickets about RC522 but I dont know if this is still compatible with the miniap and onoff shim.
And as the RC522 is mostly mentioned in bug tickets I am not sure at all how this will work in the end.

Hope someone could shine a little light on the "remove to stop" feature and what reader I need for this to work.

Happy phonieboxing

**Comment by steveleader** [comment, marked as answer]
i was in exactly the same situation some time ago and applied the hack described in the wiki for the usb reader
https://github.com/MiczFlor/RPi-Jukebox-RFID/wiki/Stop-on-removal-with-USB-RFID-Reader
works perfectly for me and my 2-year old daughter can handle it :)

**Comment by bergziege** [comment]
In the end I got a RC522 like in the manual (had to swap MISO/MOSI), run the script and it works like a charm.

Benefit of not ripping the USB device apart is that I now have a USB RFID reader to spare ;-)
